database/database_test.go

```diff
@@ -430,21 +417,18 @@ func testMarkFailed(t *testing.T) {
 
 	// insert two regular documents and one invalid one
 	db.CreateBlockedSkylink(ctx, &BlockedSkylink{
-		Skylink:        "skylink_1",
 		Hash:           HashBytes([]byte("skylink_1")),
 		Reporter:       Reporter{},
 		Tags:           []string{"tag_1"},
 		TimestampAdded: time.Now().UTC(),
 	})
 	db.CreateBlockedSkylink(ctx, &BlockedSkylink{
-		Skylink:        "skylink_2",
 		Hash:           HashBytes([]byte("skylink_2")),
 		Reporter:       Reporter{},
 		Tags:           []string{"tag_1"},
 		TimestampAdded: time.Now().UTC(),
 	})
 	db.CreateBlockedSkylink(ctx, &BlockedSkylink{
-		Skylink:        "skylink_3",
```

Comment:
We're not checking the errors on these `db.CreateBlockedSkylink` calls.

Reply:
added, if there's an error inserting the test fails so it's not always absolutely necessary imo in cases like this, but honestly I just forgot it here I think so I added it